source/fundamentals/builders/aggregates.txt

```diff
@@ -11,3 +11,501 @@ Aggregates Builders
    :class: singlecol
 
 .. _aggregates-builders:
+
+This guide provides an overview of the :java-core-api:`Aggregates </com/mongodb/client/model/Aggregates>` class
+which provides static factory methods that build :manual:`aggregation pipeline
+operators </reference/operator/aggregation/>`.
+
+.. tip::
+
+   For brevity, you may choose to import the methods of the ``Aggregates`` class
+   statically:
+
+   .. literalinclude:: /includes/fundamentals/code-snippets/builders/AggBuilders.java
+      :start-after: begin static import
+      :end-before: end static import
+      :language: java
+      :dedent:
+
+   The examples on this page assume this static import.
+
+Match
+-----
+
+Use the ``match()`` method to create a :manual:`$match </reference/operator/aggregation/match/>`
+pipeline stage that matches incoming documents against the specified
+query filter, filtering out documents that do not match.
+
+.. tip::
+
+   The filter can be an instance of any class that implements ``Bson``, but it's
+   convenient to combine with use of the :ref:`Filters <filters-builders>` class.
+
+This example creates a pipeline stage that matches all documents where the
```

Comment:
Nit: I'm used to using and seeing "The following example" opposed to "This example"

Applies to all instances.